### PR TITLE
Delete serving-tests ns after upgrade tests

### PR DESCRIPTION
**Pull request by mgencur**
Compared with previous solution this also works on OCP 4.6 (Kube 1.19).

### CODE REVIEW

File: test/lib.bash [hunk 1]
```diff
@@ -221,8 +221,8 @@ function run_rolling_upgrade_tests {
     --resolvabledomain \
     --https
 
-  # Delete the leftover services.
-  oc delete ksvc --all --cascade='foreground' -n serving-tests
+  # Delete the leftover namespace with services.
+  oc delete namespace serving-tests
```
Comment (markusthoemmes) [inline test/lib.bash:225]: So we keep iterating on this: Why is there services left here in the first place? Is some test not doing what it should?

Comment (mgencur) [reply test/lib.bash:225]: This is how upstream Serving tests work. They start the services in PreUpgrade phase and don't delete them in PostUpgrade phase. Maybe the PostUpgrade phase should delete them in upstream...

Comment (markusthoemmes) [reply test/lib.bash:225]: I'd think it absolutely should, yes.

Comment (mgencur) [reply test/lib.bash:225]: Alright. Can we merge this and then I can work upstream on deleting the services?

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Comment by mgencur** [comment]
@markusthoemmes Thanks, I'm already working on this upstream: https://github.com/knative/serving/pull/12395

**Pull request merged**